react.gradle

```diff
@@ -314,18 +314,24 @@ afterEvaluate {
                         from(jsBundleDir)
                     }
                 } else {
-                    into ("assets/${targetPath}") {
+                    // Workaround for Android Gradle Plugin 7.1 asset directory
+                    if (androidComponents.pluginVersion.major == 7 && androidComponents.pluginVersion.minor == 1) {
```

Comment:
I'm not entirely sure we need a if-then-else. We can just add another `into` directive. I believe other folders will be ignored (or at the end of the day, you'll end up having just one bundle file regardless).

Reply:
Makes sense, I've just removed it